packages/shared/src/utils/contract/trade-url-params-config.ts

```diff
@@ -68,7 +68,10 @@ export const getTradeURLParams = ({ active_symbols = [], contract_types_list = {
                 contract_types_list[key]?.categories || [];
             return [...acc, ...categories.map(contract => (contract as TTextValueStrings).value)];
         }, []);
-        const isTradeTypeValid = contractList.includes(trade_type ?? '');
+
+        const isTradeTypeValid =
+            (contractList.length === 0 && trade_type !== '') ||
+            (contractList.length > 0 && contractList.includes(trade_type ?? ''));
```

Comment:
This will fix only V1 issue or both (V1+2)? 
Also will the code below works? It's absolutely optional suggestion 
```
        const isTradeTypeValid =
            (!contractList.length && trade_type) ||
            (contractList.length && contractList.includes(trade_type ?? ''));
```

Reply:
For now I focused on V1 . I think need to handle it differently for V2